src/shared/services/bolt/bolt.ts

```diff
@@ -37,13 +37,14 @@ import {
 import { NATIVE } from 'services/bolt/boltHelpers'
 import { setupBoltWorker, addTypesAsField } from './setup-bolt-worker'
 
-import BoltWorkerModule from 'worker-loader?inline!./boltWorker.js'
+// @ts-expect-error ts-migrate(2307) FIXME: Cannot find module 'worker-loader?inline!./boltWor... Remove this comment to see the full error message
+import BoltWorkerModule from 'worker-loader?inline!./boltWorker'
```

Comment:
What is going on here. Why do we do worker-loader?inline?

Reply:
I'm not sure, maybe we didn't use webpack when this line was written, it should be possible to move the loader to the webpack config.